app/src/main/java/com/infomaniak/swisstransfer/ui/screen/newtransfer/importfiles/components/TransferOption.kt

```diff
@@ -38,21 +38,21 @@ import com.infomaniak.swisstransfer.ui.theme.SwissTransferTheme
 import com.infomaniak.swisstransfer.ui.utils.PreviewLightAndDark
 
 @Composable
-fun TransferAdvancedSetting(
-    settingType: TransferAdvancedSettingType,
+fun TransferOption(
```

Comment:
Shouldn't this class be `TransferOptionType` ? As it represent one item of  `TransferOptionsTypes`